packages/client/src/forms/configuration/index.ts

```diff
@@ -384,11 +386,20 @@ export function configureRegistrationForm(
         /*
          * Splice the disabled field away only when in registration, not in configuration mode
          */
-        newForm.sections[
-          defaultPropCustomisation.defaultField.selectedSectionIndex
-        ].groups[
-          defaultPropCustomisation.defaultField.selectedGroupIndex
-        ].fields.splice(defaultPropCustomisation.defaultField.index, 1)
+
```

Comment:
Hidden fields were not hiding anymore because this block had moved after repositioning custom fields